src/Engine/ProtoCore/Lang/BuiltInMethods.cs

```diff
@@ -475,7 +494,8 @@ public BuiltInMethods(Core core)
                         new KeyValuePair<string, ProtoCore.Type>("array", TypeSystem.BuildPrimitiveTypeObject(PrimitiveType.kTypeVar, Constants.kArbitraryRank)),
                         new KeyValuePair<string, ProtoCore.Type>("member", TypeSystem.BuildPrimitiveTypeObject(PrimitiveType.kTypeVar, 0))
```

Comment:
Rename the parameter name "member" to "element", and the description string to ChecksIfArrayContainsTheElement